openvdb_ax/openvdb_ax/compiler/Logger.h

```diff
@@ -130,7 +133,9 @@ class OPENVDB_AX_API Logger
     bool getWarningsAsErrors() const;
 
     /// @brief Sets the maximum number of errors that are allowed before
-    ///   compilation should exit
+    ///   compilation should exit.
+    /// @note The logger will continue to increment the error counter beyond
+    ///   this value but, once reached, it will not invoke the error callback.
```

Comment:
as above, I don't know if it should do this.

Reply:
Replied above